components/tikv_util/src/trace.rs

```diff
@@ -45,3 +59,67 @@ pub fn encode_spans(span_sets: Vec<SpanSet>) -> impl Iterator<Item = spanpb::Spa
         })
         .into_iter()
 }
+
+#[cfg(test)]
+mod tests {
+    use kvproto::span as spanpb;
+
+    #[test]
+    fn test_encode_spans() {
+        let (guard, collector) = minitrace::trace_enable(0u32);
+        let guard2 = minitrace::new_span(1u32);
+        std::mem::drop(guard2);
+        std::mem::drop(guard);
+
```

Comment:
Could you please construct `Vec<minitrace::SpanSet>` handly to be encoded, then compare the encoded `Vec<spanpb::SpanSet>` with another `Vec<spanpb::SpanSet>` constructed handly?